src/validation/index.js

```diff
@@ -25,6 +25,19 @@ const allFieldsRequired = (res, message) => {
   });
 };
 
+/**
+ * Defines the failed message returned when required fields are missing.
+ * @param {object} res - Response object
+ * @param {string} message - specific error message
+ * @returns {res} - Response object
+ */
+const errorMsg = (res, message) => {
+  res.status(400).send({
+    success: false,
+    message,
+  });
```

Comment:
This method already exists. Please do use the `clientError` method in the `utils/helperMethod.js` file.

Reply:
done thanks